File: BPMOPMobile_VietNamAirLines/BPMOPMobile_VietNamAirLines/BPMOPMobileV1.iOS/Model/ViewBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Foundation;
using UIKit;

namespace BPMOPMobileV1.Model
{
    public class ViewBase
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Value { get; set; }
        public bool Enable { get; set; }
        public bool IsRequire { get; set; }
    }
}

File: VNABPMOP/app/src/main/java/com/vuthao/bpmop/base/model/dynamic/ViewBase.java
package com.vuthao.bpmop.base.model.dynamic;

public class ViewBase {
    private String ID;
    private String Title;
    private String Value; // string
    private boolean Enable;
    private boolean IsRequire;

    public ViewBase() {
    }

    public String getID() {
        return ID;
    }

    public void setID(String ID) {
        this.ID = ID;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getValue() {
        return Value;
    }

    public void setValue(String value) {
        Value = value;
    }

    public boolean isEnable() {
        return Enable;
    }

    public void setEnable(boolean enable) {
        Enable = enable;
    }

    public boolean isRequire() {
        return IsRequire;
    }

    public void setRequire(boolean require) {
        IsRequire = require;
    }
}
